brainglobe_atlasapi/atlas_generation/atlas_scripts/kim_devccf_mouse.py

```diff
@@ -0,0 +1,242 @@
+import json
+import time
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pooch
+from brainglobe_utils.IO.image import load_nii
+from rich.progress import track
+
+from brainglobe_atlasapi import utils
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.config import DEFAULT_WORKDIR
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+ATLAS_NAME = "kim_devccf_mouse"  # multiple versions of the same atlas
+SPECIES = "Mus musculus"
+ATLAS_LINK = "https://kimlab.io/brain-map/DevCCF/"
+CITATION = "Kronman, F.N., Liwang, J.K., Betty, R. et al. 2024, https://doi.org/10.1038/s41467-024-53254-w"
+ORIENTATION = ["left", "superior", "posterior"]
+ROOT_ID = 15564
+RESOLUTION_UM = 20
+VERSION = 1
+PACKAGER = "Carlo Castoldi <castoldi[at]ipmc.cnrs.fr>"
+ATLAS_FILE_URL = "https://doi.org/10.6084/m9.figshare.26377171.v1"
+
+TIMEPOINTS = ("E11.5", "E13.5", "E15.5", "E18.5", "P04", "P14", "P56")
+RESOLUTIONS = (20, 50)
+MODALITIES = (
+    "LSFM",  # Light Sheet Fluorescence Microscopy
+    "MRI-adc",  # MRI Apparent Diffusion Coefficient
+    "MRI-dwi",  # MRI Difusion Weighted Imaging
+    "MRI-fa",  # MRI Fractional Anisotropy
+    "MRI-MTR",  # MRI Magnetization Transfer Ratio
+    "MRI-T2",  # MRI T2-weighted
+)
+
+
+def pooch_init(download_dir_path: Path):
+    utils.check_internet_connection()
+
+    hash = None
+    registry = {a + ".zip": hash for a in TIMEPOINTS}
+    registry["DevCCFv1_OntologyStructure.xlsx"] = hash
```

Comment:
It would be great if these hashes were the correct ones instead of `None`. Can be addressed separately as part of #442 if you prefer (but would be amazing if you wanted to take time for this!)